Source/cmake/OptionsWPE.cmake

```
@@ -197,6 +197,16 @@ if (USE_SYSTEMD)
     endif ()
 endif ()
 
+if (ENABLE_BREAKPAD)
+    find_package(Breakpad REQUIRED)
+    add_definitions(-DUSE_BREAKPAD=1)
```

Maybe use something like `WEBKIT_OPTION_DEFINE(ENABLE_BREAKPAD, ....)` before this `if` to declare `ENABLE_BREAKPAD`, exposing it to the build and avoiding having to define `USE_BREAKPAD`?

```suggestion
WEBKIT_OPTION_DEFINE(ENABLE_BREAKPAD, "Whether or not enable breakpad minidump support." PUBLIC OFF)
if (ENABLE_BREAKPAD)
    find_package(Breakpad REQUIRED)
```